addons/resource/models/resource_calendar_attendance.py

```diff
@@ -64,6 +70,16 @@ def get_week_type(self, date):
         # some years have 53 weeks. Therefore, two consecutive odd week number follow each other (53 --> 1).
         return int(math.floor((date.toordinal() - 1) / 7) % 2)
 
+    @api.depends('hour_from', 'hour_to')
+    def _compute_duration_hours(self):
+        for attendance in self:
+            attendance.duration_hours = (attendance.hour_to - attendance.hour_from) if attendance.day_period != 'lunch' else 0
```

Comment:
Might be correct, but I'm wondering if there aren't some case where knowing the duration of the lunch might be useful, for example in cases of overtime duration computing etc.

Reply:
My understanding of lunch periods is that they were added as a way to to have a contiguous period without some of it being counted as working time, basically having duration 0.